WebRoot/js/zimbraMail/package/DocsPreview.js

```diff
@@ -25,7 +25,7 @@ AjxPackage.require("ajax.util.AjxStringUtil");
 AjxPackage.require("ajax.util.AjxTimedAction");
 AjxPackage.require("ajax.util.AjxText");
 
-AjxPackage.require("ajax.util.AjxTimezoneData");
+//AjxPackage.require("ajax.util.AjxTimezoneData");
```

Comment:
it is included from jsp file, we can't package this file with other files as it can be updated at runtime